src/main/java/Deadlines.java

```diff
@@ -0,0 +1,23 @@
+public class Deadlines extends Task {
+    private String endTime;
+    private String taskLabel = "[D]";
+    public Deadlines (String input){
+        super(input.substring(9,input.indexOf('/') - 1)); // Sanitize input by removing "deadline" at the start
```

Comment:
Personally, maybe you can replace the number `9` with a constant to avoid "magic numbers"?

> Like `private static final int DEADLINE_LABEL = 9;`?

Reply:
Agreed. Thanks for the idea!